hushline/model.py

```diff
@@ -349,3 +411,96 @@ def __init__(self) -> None:
 
     def __repr__(self) -> str:
         return f"<InviteCode {self.code}>"
+
+
+# Paid tiers
+class Tier(Model):
+    __tablename__ = "tiers"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    name: Mapped[str] = mapped_column(db.String(255), unique=True)
+    monthly_amount: Mapped[int] = mapped_column(db.Integer)  # in cents USD
+    stripe_product_id: Mapped[Optional[str]] = mapped_column(db.String(255), unique=True)
+    stripe_price_id: Mapped[Optional[str]] = mapped_column(db.String(255), unique=True)
+
+    def __init__(self, name: str, monthly_amount: int) -> None:
+        super().__init__()
+        self.name = name
+        self.monthly_amount = monthly_amount
+
+
+class StripeEvent(Model):
+    __tablename__ = "stripe_events"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    event_id: Mapped[str] = mapped_column(db.String(255), unique=True, index=True)
+    event_type: Mapped[str] = mapped_column(db.String(255))
+    event_created: Mapped[int] = mapped_column(db.Integer)
+    event_data: Mapped[str] = mapped_column(db.Text)
+    status: Mapped[Optional[StripeEventStatusEnum]] = mapped_column(
+        SQLAlchemyEnum(StripeEventStatusEnum), default=StripeEventStatusEnum.PENDING
+    )
+    error_message: Mapped[Optional[str]] = mapped_column(db.Text)
+
+    def __init__(self, event: Event) -> None:
+        super().__init__()
```

Comment:
Pass args as `kwargs` in `super().__init__(...)`

Reply:
Fixed in 40b1ca102ea381b325f06f6a1ec5ceb61c4228a2